### PR TITLE
Add support for BSM on Case and on Correspondence Parties

### PR DESCRIPTION
**Pull request by dkumanov**

### CODE REVIEW

File: version1/02 - Object types.md [hunk 1]
```diff
@@ -258,29 +258,31 @@ The following notation is used in the tables below:
...
+| M409      | string    | land                          |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M410      | string    | epostadresse                  |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M411      | string    | telefonnummer                 |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M412      | string    | kontaktperson                 |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+|           |           |                               |          |             |           |  |      |      |       |      |      |        |                                        |
+|           | object    | virksomhetsspesifikkeMetadata |          |             |           |  |  x   |  x   |       |      |      |        |                                        |
```
Comment (teodoradobreva) [inline version1/02 - Object types.md]: Field no. should be M711 here, for Sakspart only, as it is defined in Noark 5.5.

Comment (dkumanov) [reply version1/02 - Object types.md]: Added in 58c9093.

**Review by teodoradobreva — APPROVED** [review]

**Review by orlinDocumaster — APPROVED** [review]

**Comment by dkumanov** [comment]
Rebased on the latest develop. No other changes.

**Comment by dkumanov** [comment]
Squashed 2 commits into one in preparation for merge.

**Pull request merged**